test/blocks/media/media.test.js

```diff
@@ -67,6 +68,33 @@ describe('media', () => {
       const appStoreCta = medias[5].querySelector('a.app-store');
       expect(appStoreCta).to.exist;
     });
+    it('desktop view has visibile qr code image and no google-play and app-store CTA', async () => {
+      await setViewport({ width: 1200, height: 100 });
+      const qrCodeImg = medias[5].querySelector('img.qr-code-img');
+      const qrCTA = medias[5].querySelectorAll('.qr-button-container');
+      expect(window.getComputedStyle(qrCodeImg).getPropertyValue('display')).not.to.be.equal('none');
```

Comment:
nit: do you need the `be` in the matchers here and below? Can it be:
`expect(window.getComputedStyle(qrCodeImg).getPropertyValue('display')).not.to.equal('none')`

Reply:
Thanks for the review, @JasonHowellSlavin Both worked but I have updated to remove .be